### ISSUE TITLE
Sound block stops counting after sound reaches 100%

### ISSUE THREAD

**Issue by droslean** [greshake/i3status-rust#266]

**Comment by atheriel** [comment]
So the block is not reporting volume over 100%, even though you can set your volume higher than that on your system (via the volume button, I'm guessing) -- is that right?

**Comment by droslean** [comment]
For example, if I start decreasing the volume by 12% each time, it will go like this
`12% -> 24% ->......-> 84% -> 96% -> 108%
And it will stay at 108%, despite the fact I can continue increasing the volume.

**Comment by droslean** [comment]
That will happen if I start changing the volume manually and not by using the `MouseButton::WheelUp`

**Comment by ammgws** [comment]
```rust
MouseButton::WheelUp => {

if volume < 100 {

device.set_volume(

min(self.step_width, 100 - volume) as i32,

)?;
}
```

Do we really need to bound it at 100% if there's no such limitation on the amixer side?

**Comment by atheriel** [comment]
I'm in favour of limiting the input from the mouse wheel to 0-100 because I'm worried about sending nonsense signals to `amixer` (or the sound card itself). However, we could **display** values outside that range if that's really what `amixer` tells us the volume is.

**Comment by ammgws** [comment]
OK so just as a note `amixer` doesn't let you set the volume above 100%, however `pactl` does (although with distortion). I've been using pactl to increase/decrease volume and never go near 100% so didn't notice the bar wasn't displaying the actual volume. So I think your suggestion is good, unless people want to be able to define what their max volume is in the block config.

```sh
>> amixer set Master 999%
Simple mixer control 'Master',0
  Capabilities: pvolume pswitch pswitch-joined
  Playback channels: Front Left - Front Right
  Limits: Playback 0 - 65536
  Mono:
  Front Left: Playback 65536 [100%] [on]
  Front Right: Playback 65536 [100%] [on]
>> pactl set-sink-volume @DEFAULT_SINK@ 999%
>> amixer get Master
Simple mixer control 'Master',0
  Capabilities: pvolume pswitch pswitch-joined
  Playback channels: Front Left - Front Right
  Limits: Playback 0 - 65536
  Mono:
  Front Left: Playback 654704 [999%] [on]
  Front Right: Playback 654704 [999%] [on]
>> amixer set Master 10%+
Simple mixer control 'Master',0
  Capabilities: pvolume pswitch pswitch-joined
  Playback channels: Front Left - Front Right
  Limits: Playback 0 - 65536
  Mono:
  Front Left: Playback 65536 [100%] [on]
  Front Right: Playback 65536 [100%] [on]
```

**Comment by droslean** [comment]
So, shall we add in the `sound` block, the option to set the limit? 
But either way, if the user assigns his `xbindkeys` keys to increase the volume, using either `amixer` ,  `pactl` or `ponymix` ,  100% won't limit him.  In `ponymix` there is a parameter to set the max volume but `amixer` can't do that. 
So, in conclusion,  we should not limit the user in 100% using the `MouseWheel` and add a `limit` option to the `block`. The `xbindkeys` are not related to the status bar,  and if the user wants to increase his volume more than 100% he should manage his limits. But the status bar can manage the limits if using `MouseWheel` .

**Comment by ammgws** [comment]
The `limit` option would only change the max displayed volume - the max volume settable with the mouse wheel will still be capped at 100% since we're using `amixer`. Is that what you are envisioning?

**Comment by droslean** [comment]
amixer doesn't set any limit

**Comment by ammgws** [comment]
So you get a different result if you run the same commands as I did a couple comments above?

**Comment by droslean** [comment]
Yes. the status bar stops reporting the volume after it reaches 100%.
